src/components/map/Map.tsx

```diff
@@ -20,29 +19,41 @@ export default function Map() {
   const buckets = getBuckets(mapRecords.features);
 
   useEffect(() => {
-    // Factor in "include_in_n" for population unfiltered geography estimates
-    const must_include_in_n = state.filters.population_group.size === 0;
-    const prevalenceCountryDict: Record<string, AggregatedRecord> = getAggregateData(state.filtered_records, AggregationFactor.country, must_include_in_n)
-      .reduce((a, x: AggregatedRecord) => ({ ...a, [x.name]: x }), {})
-
-    const importGeo = Countries as any;
-    const features = importGeo.features as GeoJSON.Feature[]
-
-    // We will iterate through all the features in the geoJson
-    // if they are in the country dict we will attach their aggregated data to the feature for displaying
-    importGeo.features = features.map(feature => {
-      const country = prevalenceCountryDict[feature?.properties?.name];
-      if (country && country.seroprevalence) {
-        const { seroprevalence, error, n, num_studies } = country;
-        return { ...feature, properties: { ...feature.properties, seroprevalence, error, n, num_studies } }
+    if(state.country_prevalences.length > 0){
+      const updateMap = async () => {
```

Comment:
Why is this async I don't see any awaits?

Reply:
yeah will change this, sorry this was leftover from a previous implementation